### PR TITLE
Solution to run master branch on docker

### PR DESCRIPTION
**Pull request by woutersamaey**
@NicolasDorier for your review.

### CODE REVIEW

**Comment by pavlenex** [comment]
Can you ELI5 what is this and why it's useful? Seems very edge-case to be documented in contribute docs.

**Comment by woutersamaey** [comment]
I built this with Nicolas help and he told me to put it here.

This script reconfigures your docker install to pull. build and run the latest code from master.

Nicolas and I will be using it to keep a server running using the latest code for automated testing. In Nicolas' case it will be for future Selenium tests. In my case it will be for Zapier.

**Comment by pavlenex** [comment]
I see. Sorry my bad. I somehow misread that this is doc repository hence the question.

**Comment by NicolasDorier** [comment]
No need of having several fragments. Only one is enough.

The readme should have more explanation about what it is about.

I think the script is not really needed, you can just document the copy of the fragment and the `btcpay-setup.sh` in a code block, in the readme.

Cronjob shouldn't be mentioned. Nothing is related to cron in this, people run it as they see fit.
